docs/src/maintain/manage-releases.md

```diff
@@ -13,55 +13,61 @@ Releases are when a project formally publishes a new version so the community ca
 * Regular releases that follow [semantic versioning](https://semver.org/) and are considered production-ready.
 * Prereleases that are not considered production-ready and are intended to give the community a preview of upcoming changes.
 
-## Release Team
+## Release Manager
 
-A two-person release team is assigned to each scheduled release. This two-person team is responsible for:
+One member of the Technical Steering Committee (TSC) is assigned to manage each scheduled release. The release manager is determined at the TSC meeting the day before the release.
+
+The release manager is responsible for:
 
 1. The scheduled release on Friday
 1. Monitoring issues over the weekend
 1. Determining if a patch release is necessary on Monday
 1. Publishing the patch release (if necessary)
 
-The two-person team should seek input from the whole team on the Monday following a release to double-check if a patch release is necessary.
+The release manager should seek input from the whole team on the Monday following a release to double-check if a patch release is necessary.
 
-At least one member of the release team needs to have access to eslint's two-factor authentication for npm in order to do a release.
+The release manager needs to have access to ESLint's two-factor authentication for npm in order to do a release.
 
 ## Release Communication
 
 Each scheduled release should be associated with a release issue ([example](https://github.com/eslint/eslint/issues/8138)). The release issue is the source of information for the team about the status of a release. Be sure the release issue has the "release" label so that it's easy to find.
 
 ## Process
 
-On the day of a scheduled release, the release team should follow these steps:
+On the day of a scheduled release, the release manager should follow these steps:
 
 1. Review open pull requests to see if any should be merged. In general, you can merge pull requests that:
-    * Have been open at least two days and have been reviewed (these are just waiting for merge).
+    * Have been open for at least two days and approved (these are just waiting for merge).
     * Important pull requests (as determined by the team). You should stop and have people review before merging if they haven't been already.
     * Documentation changes.
     * Small bugfixes written by a team member.
 1. Log into Jenkins and schedule a build for the "ESLint Release" job.
 1. Watch the console output of the build on Jenkins. At some point, the build will pause and a link will be produced with an input field for a six-digit 2FA code.
 1. Enter the current six-digit 2FA code from your authenticator app.
 1. Continue the build and wait for it to finish.
-1. Update the release blog post with a "Highlights" section, including new rules and anything else that's important.
+1. Update a release blog post with a "Highlights" section, including new rules and anything else that's important.
```

Comment:
```suggestion
1. Update the release blog post with a "Highlights" section, including new rules and anything else that's important.
```

I think "the" makes more sense here as this refers to the blog post created during the previous step.